lib/sww.js

```diff
@@ -52,13 +53,28 @@ ServiceWorkerWare.prototype.handleEvent = function sww_handleEvent(evt) {
 
 ServiceWorkerWare.prototype.onFetch = function sww_onFetch(evt) {
   var steps = this.router.match(evt.request.method, evt.request.url);
+
+  // Push the fallback middleware at the end of the list if it's defined
+  // also being sure that wont be executed if there is a response already.
+  if (this.fallbackMw) {
+    var fallback = this.fallbackMw;
+    steps.push(function(req, res) {
```

Comment:
Don't force the pattern here. Because you don't know about how the default middleware is implemented. Maybe it wants to attend the request even if it is already defined. Suppose the case I want to finally add a special header marking the response with `X-SWW-Version: 0.1.14`.